test/ServerSpec.js

```diff
@@ -0,0 +1,33 @@
+var expect = require('chai').expect;
+var request = require('request');
+
+
+describe('', function() {
+
+  describe('User creation', function() {
+    var signUpOptions = {
+      method: 'POST',
+      url: 'http://localhost:3000/api/signup',
+      json: {
+        username: 'Phillip',
+        password: 'Phillip'
+      }
+    };
+
+    it('Should add a new user to the database', function(done) {
```

Comment:
Yay for tests! Should we rename this to "Should return user info in response body for successful signups" ? Does it make sense to check that we actually inserted documents into the Mongo database?